### PR TITLE
python-magic 0.4.27 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/p/python-magic.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class PythonMagic < Formula
+  desc "Python wrapper for libmagic"
+  homepage "https://github.com/lxml/lxml"
```
Comment (singingwolfboy) [inline Formula/p/python-magic.rb:3]: Please update the homepage.

File: Formula/p/python-magic.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  url "https://files.pythonhosted.org/packages/da/db/0b3e28ac047452d079d375ec6798bf76a036a08182dbb39ed38116a49130/python-magic-0.4.27.tar.gz"
+  sha256 "c1ba14b08e4a5f5c31a302b7721239695b2f0f058d125bd5ce1ee36b9d9d3c3b"
+  license all_of: ["BSD-2-Clause", "MIT"]
+
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.12" => [:build, :test]
```
Comment (singingwolfboy) [inline Formula/p/python-magic.rb:10]: Needs a dependency on `libmagic`

**Review by singingwolfboy — CHANGES_REQUESTED** [review]

File: Formula/p/python-magic.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/p/python-magic.rb]: make sense! 😅

File: Formula/p/python-magic.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/p/python-magic.rb:10]: just added 👍

**Review by p-linnane — APPROVED** [review]

**Review by singingwolfboy — APPROVED** [review]

**Pull request merged**